specification/_types/query_dsl/specialized.ts

```diff
@@ -201,3 +201,15 @@ export class ShapeFieldQuery {
   relation?: GeoShapeRelation
   shape?: GeoShape
 }
+
+/**
+ * @variants container
+ */
+export class RuleQuery extends QueryBase {
```

Comment:
What is the purpose of this type ? It doesn't seem to be used anywhere.

Reply:
Thanks for the question. I'd added it because it's a [new part of the query DSL](https://github.com/elastic/elasticsearch/pull/97250). If it's unnecessary I can remove it. What are your thoughts?

Reply:
Per discussion, I have removed the container from this object, and added the rule query to the container object. Thank you!